conf/cassandra.yaml

```diff
@@ -1804,6 +1805,15 @@ drop_compact_storage_enabled: false
 # write_consistency_levels_warned: []
 # write_consistency_levels_disallowed: []
 #
+# Guardrail to warn or fail when writing partitions larger than threshold.
+# The guardrail is only checked when writing sstables (flush, compaction, streaming, etc.), and exceeding the fail
+# threshold on that moment will only log an error message, without interrupting the operation.
+# This operates on a per-sstable basis, so it won't detect a large partition if it is spread across multiple sstables.
+# The warning threshold replaces the deprecated config property compaction_large_partition_warning_threshold.
+# The two thresholds default to null to disable.
```

Comment:
Can you add a few valid examples to see syntax/units? 100MiB, 100MB, 10KB,...

Reply:
My question is : should we regardless of the stream situation? Because the stream data is generated by flush and compaction or logging is enough ?

Reply:
These sstables that are going to stream to other nodes are generated by flush or compaction , and we may just need to use the guardrail to detecte at flush or compaction stage. 

The reveiver  or the sending node do not need to use the guardrail to detecte at stream stage. 

That is what I mean.

Reply:
Thx on the examples.

Reply:
> These sstables that are going to stream to other nodes are generated by flush or compaction , and we may just need to use the guardrail to detecte at flush or compaction stage.

I see, I have updated the comment to exclude streaming.